pallets/account-linker/Cargo.toml

```diff
@@ -25,13 +25,17 @@ sp-io = { default-features = false, version = '2.0.1' }
 sp-std = { default-features = false, version = '2.0.1' }
 sha2 = { default-features = false, version = "0.9.2" }
 ripemd160 = {default-features = false, version = "0.9.1" }
+sp-runtime = { default-features = false, version = '2.0.1', optional = true }
+frame-benchmarking = {default-features = false, version = '2.0.1', optional = true }
+sp-core = {default-features = false,  version = '2.0.1', optional = true }
 
 [dev-dependencies]
 sp-core = { default-features = false, version = '2.0.1'}
```

Comment:
Seems then some of these doesn't need to be in dev deps?

Reply:
removed from dev deps.